services/QuillLMS/client/app/bundles/Staff/components/evidence/configureRules/ruleHint.tsx

```diff
@@ -69,19 +63,18 @@ const RuleHint = ({
         <p className="form-subsection-label">Hint Image Alt Text</p>
         <textarea
           aria-label="Hint Image Alt Text"
-          className="image-attribution-text-area"
+          className="image-alt-text-area"
           onChange={handleHintAltTextChange}
           value={hint.image_alt_text}
         />
-        <button className="remove-highlight quill-button small secondary outlined" onClick={handleRemoveHint} type="button">Remove Hint</button>
       </React.Fragment>
     )
   }
 
   return(
     // eslint-disable-next-line
     <React.Fragment>
-      {hint && !hint._destroy ? renderHint() : <button className="add-feedback-layer quill-button small primary outlined" onClick={handleHintCreation} type="button">Add Hint</button>}
+      {renderHint()}
     </React.Fragment>
```

Comment:
NBD but this doesn't need to be nested in a Fragment anymore, and I don't think that `eslint-disable` is necessary any longer.

Reply:
@emilia-friedberg I tried removing this wrapper, and started getting errors on render: `Uncaught Error: Objects are not valid as a React child (found: object with keys {}). If you meant to render a collection of children, use an array instead.`.  There may be a better way to get that error to stop happening besides wrapping the return value in a `React.Fragment>` element, but that did work.  Any suggestions for a better solution?